x-pack/plugin/core/template-resources/src/main/resources/agentless@settings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "template": {
+    "settings": {
+      "index": {
+        "hidden": true,
+        "number_of_shards": 1,
+        "number_of_replicas": 1
```

Comment:
nitpick: I would suggest to omit these (shard/replica) as they are the defaults.